setup.py

```diff
@@ -26,11 +26,9 @@
     python_requires=">=3.7",
     install_requires=[
         "flatten-dict==0.4.2",
-        "jupyter==1.0.0",
         "matplotlib>=3.5.3",
```

Comment:
Why is `matplotlib` required?

Reply:
Tests were failing, fixed it. `matplotlib` is optional now.